website/blog/2024-03-07-release-1.8.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Podman Desktop 1.8 Release
+description: Podman Desktop 1.8 has been released!
+slug: podman-desktop-release-1.8
+authors: deboer
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.8/selkie-family.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.8 Release! 🎉
+
+![Podman-desktop-1-8-hero](img/podman-desktop-release-1.8/selkie-family.png)
+
+We've got a new release with a ton of seal appeal! This release introduces:
+
+- **Podman 4.9.3**: [Podman 4.9.3](https://github.com/containers/podman/releases) is now included in both Windows and Mac installers.
```

Comment:
Can't we link to the 4.9.3 release directly ?

Reply:
Not sure why we didn't before (one less thing to change each time?) but I did now.